src/elements/Input/Input.js

```diff
@@ -97,6 +98,10 @@ const Input = ({
 	}
 
 	if(type === 'textarea') {
+		if (checkSensitiveInfo) {
```

Comment:
Let's _not_ have this check. We shouldn't allow our inputs to save this type of data _except_ for the specific social security number input we have for our credit features (see this: https://github.com/the-control-group/intelius-dashboard/blob/main/src/idp/components/CreditCheckForm.js#L118)

And for this case, we should actually create a specific SSN input within the UI to handle this exception.